README.md

```diff
@@ -65,9 +119,32 @@ cargo test -p wgpu -- <name-of-test>
 
 If you are a user and want a way to help contribute to wgpu, we always need more help writing test cases. 
 
+### WebGPU Conformance Test Suite
+
+WebGPU includes a Conformance Test Suite to validate that implementations are working correctly. We can run this CTS against wgpu.
+
+To run the CTS, first you need to check it out:
+
+```
+git clone https://github.com/gpuweb/cts.git
+cd cts
+git checkout a44186d9fa1d5babdb4a198e8ef04cf0d7035ebb
```

Comment:
can we refer to the CI script for the revision? Alternatively, we can store the revision in `cts_runner/revision.txt` and just use it both here and in the CI script.